testing.rtf

```diff
@@ -0,0 +1 @@
+Testing file.
```

Comment:
Looks like this guy snuck in here, should be removed.

Reply:
Those sneaky rtf files, I'll delete it.